conans/test/functional/scm/scm_test.py

```diff
@@ -38,6 +42,10 @@ def build(self):
 base_svn = base % "svn"
 
 
+def _quoted(item):
```

Comment:
I don't get this ``_quoted()`` function. It seems it was ok with the ``scm`` attribute quoting it. If it is related to the ``svn.get_remote_url()`` already returned a quoted string, it seems that is that one the one that should get fixed.

Reply:
It is about the key `url` inside the `scm` attribute:

```python
scm = {{
   "type": "%s",
   "url": {url},
   "revision": "{revision}",
}}
```

Sometimes it should be replaced with `"url"` (quoted string) and others with `get_remote_url()` without quotes. I found that `_quoted` function cleaner than having `"\"auto\""` for the replacements. I can change it back, no problem, just tell me.

Reply:
Ok, good